packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx

```diff
@@ -1,20 +1,27 @@
 import { IconButton } from '@rocket.chat/fuselage';
 import type { FC } from 'react';
+import { forwardRef } from 'react';
 
-const ToolBoxAction: FC<any> = ({ id, icon, color, action, className, index, title, 'data-tooltip': tooltip, ...props }) => (
-	<IconButton
-		data-qa-id={`ToolBoxAction-${icon}`}
-		className={className}
-		onClick={() => action(id)}
-		data-toolbox={index}
-		key={id}
-		icon={icon}
-		position='relative'
-		tiny
-		overflow='visible'
-		color={!!color && color}
-		{...{ ...props, ...(tooltip ? { 'data-tooltip': tooltip, 'title': '' } : { title }) }}
-	/>
-);
+const ToolBoxAction: FC<any> = forwardRef(function ToolBoxAction(
```

Comment:
Why do you need the `ref` here?

Reply:
I need the `reference`  to send to the `Dropdown` component to its position correctly.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/27579/files#diff-818a123e5e5c1d53e9de49e0a9371760a63314456a1c6306c663d8af8c2f300bR32

Reply:
got it